docs/versioned-plugins/filters/ruby-v3.1.0.asciidoc

```diff
@@ -182,11 +182,20 @@ The path of the ruby script file that implements the `filter` method.
 [id="{version}-plugins-{type}s-{plugin}-script_params"]
 ===== `script_params`
 
-  * Value type is <<hash,hash>>
+  * Value type is {logstash-ref}/configuration-file-structure.html#hash[hash]
   * Default value is `{}`
 
 A key/value hash with parameters that are passed to the register method
 of your ruby script file defined in `path`.
 
+[id="{version}-plugins-{type}s-{plugin}-tag_on_exception"]
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin. We can fix it by adding the missing content to all the relevant files.